lib/msal-browser/src/internals.ts

```diff
@@ -24,6 +25,7 @@ export { NativeInteractionClient } from "./interaction_client/NativeInteractionC
 export { RedirectHandler } from "./interaction_handler/RedirectHandler";
 export { EventHandler } from "./event/EventHandler";
 export { NativeMessageHandler } from "./broker/nativeBroker/NativeMessageHandler";
+export { ResponseHandler } from "@azure/msal-common";
```

Comment:
Is this to avoid `common` exports in the future?

Reply:
I don't really want to add new stuff to internals, let's instead focus on what we need to do to remove internals all-up